Makefile

```diff
@@ -72,6 +69,9 @@ install-docs: docs/skopeo.1
 	install -d -m 0755 ${MANINSTALLDIR}/man1
 	install -m 644 docs/skopeo.1 ${MANINSTALLDIR}/man1/
 
+install-completions:
+	install -m 644 -T hack/make/bash_autocomplete ${BASHINSTALLDIR}/skopeo
```

Comment:
I already fixed this.  @mikebrow in a build system no directories pre-exist.  So this was blowing up rpmbuild.